src/server/scripts/Outland/BlackTemple/boss_shade_of_akama.cpp

```diff
@@ -157,236 +210,148 @@ class boss_shade_of_akama : public CreatureScript
 
     struct boss_shade_of_akamaAI : public BossAI
     {
-        boss_shade_of_akamaAI(Creature* creature) : BossAI(creature, DATA_SHADE_OF_AKAMA)
-        {
-            Initialize();
-            me->SetFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NOT_SELECTABLE);
-        }
+        boss_shade_of_akamaAI(Creature* creature) : BossAI(creature, DATA_SHADE_OF_AKAMA) { }
 
         void Initialize()
         {
-            combatStarted = false;
-            akamaReached = false;
-            HasKilledAkama = false;
-            HasKilledAkamaAndReseting = false;
+            Spawners.clear();
```

Comment:
`Initialize()` isn't a method on the base class tmk?

But yeah still bad form.